src/examples/completeExample.json

```diff
@@ -0,0 +1,23 @@
+{
+  "nodeType": "formulas",
+  "forest": [
+    {
+      "nodeType": "formulas",
+      "forest": [],
+      "formulas": {
+        "resolved": false,
+        "row": 1,
+        "value": "P&Q"
+      },
+      "rule": "&"
```

Comment:
These nodes/formulas/rules are a bit mixed up.
P&Q should be the parent node, with rule "A".
It should have a child node with 2 formulas, "P" and "Q". The child node has rule "&", because it was created by resolving the  "P&Q" node.